addons/account/models/account_payment_term.py

```diff
@@ -26,7 +26,7 @@ def _default_example_date(self):
     company_id = fields.Many2one('res.company', string='Company')
     fiscal_country_codes = fields.Char(compute='_compute_fiscal_country_codes')
     sequence = fields.Integer(required=True, default=10)
-    currency_id = fields.Many2one('res.currency', default=lambda self: self.env.company.currency_id, store=True)
+    currency_id = fields.Many2one('res.currency', compute="_compute_currency_id", default=lambda self: self.env.company.currency_id, store=False)
```

Comment:
Compute + depends doesn't make sense
`store=False` is the default for computed fields
```suggestion
    currency_id = fields.Many2one('res.currency', compute="_compute_currency_id")
```